benchmarks/benchmark_serving.py

```diff
@@ -313,7 +450,29 @@ def main(args: argparse.Namespace):
         "--version",
         type=str,
         default="N/A",
-        help="Version of the serving backend/engine.",
+        help="Version of the serving backend/engine, if known. "
+        "Only used in result json for record keeping purposes.",
+    )
+    parser.add_argument(
+        "--tp-size",
+        type=str,
+        default="N/A",
+        help="Size of tensor parallelism of the model server, if known. "
+        "Only used in result json for record keeping purposes.",
+    )
+    parser.add_argument(
+        "--pp-size",
+        type=str,
+        default="N/A",
+        help="Size of pipeline parallelism of the model server, if known. "
+        "Only used in result json for record keeping purposes.",
+    )
+    parser.add_argument(
+        "--device-name",
+        type=str,
+        default="N/A",
+        help="Device name, if known. Only used in result json for record "
+        "keeping purposes.",
     )
```

Comment:
merge these into just generic kv pairs for metadata? `--metadata pp-size=2 --metadata tp-size=4`.

Reply:
Good point! I will separate metadata from the those actually matter to the launch of the benchmark

Reply:
Updated! Now metadata can be specified like `--metadata tp=2 version=0.3.3 device-name=A100-80G`, and it will be only used if the end user is saving the benchmark result as json file.